### ISSUE TITLE
Toast doesn't show on clipboard click

### ISSUE THREAD

**Issue by achampagnedev** [liteflow-labs/liteflow-js#21]
I noticed that the success toast for copy to clipboard doesn't show. 

The console shows this error when clicking on the button: 

<img width="536" alt="image" src="https://user-images.githubusercontent.com/10590599/187375360-d9d0a8cf-e831-45fa-8606-ad3d8a6fd20b.png" />

This feature is in two components:

- [ ] Referral page `/referral` component: `components/src/Referral/Form.tsx` line 43
- [ ] Profile page `/users/*` component: `components/src/User/Profile/Info.tsx` line 68

**Comment by NicolasMahe** [comment]
Same issue on toast in `packages/components/src/Token/Form/Create.tsx`.
Got the issue on liteflow-nft-ropsten but not on demo-bsc-testnet marketplace.

**Comment by NicolasMahe** [comment]
I tried previous version of `liteflow-nft-ropsten`, specially before the switch from tsdx to tsc, and the toast are never displayed when they are trigger from the components.
Could it be related to next that import the nft libs in a different way when it's from the monorepo than from `npm install`?

**Comment by NicolasMahe** [comment]
I did some research yesterday night and I find out that we have the `@chakra-ui/react` and a 2 others (i think dependencies of chakra) dependencies in the `node_modules` of both apps and both packages components and templates:
```
apps/admin/node_modules/@chakra-ui/react
apps/test/node_modules/@chakra-ui/react
packages/components/node_modules/@chakra-ui/react
packages/templates/node_modules/@chakra-ui/react
```
Even worst, each `XXX/node_modules/@chakra-ui/react` contains a `node_modules` with more chakra deps!
I think this is causing the issue as the instance of ChakraProvider or ToastProvider may be duplicated depending where the source code lives.
On the clients' app (eg: onlyshoot) this is not the case, we have a single `@chakra-ui` folder in the root's `node_modules` containing all the chakra deps, no duplication in the `node_modules/@nft/XXX` libs:
```
node_modules/@chakra-ui/react
node_modules/@chakra-ui/...
```

I tried to solve this issue using the command `npm dedupe` without success. The goal of this command is to move to a higher level common dependencies. I think we have conflicting version of dependencies that prevent `@chakra-ui` to live in the root's `node_modules`
